.gitignore

```diff
@@ -1,5 +1,3 @@
-.
-..
-.idea/
-vendor/
-.phpunit.result.cache
\ No newline at end of file
+/vendor
+/composer.lock
+/.phpunit.result.cache
```

Comment:
. and .. have no reason to be in .gitignore
.idea belongs into global gitignore, not project gitignore
vendor and .phpunit.result.cache should only be ignored in the project root directory
composer.lock should not be committed by libraries